lib/test_suite.py

```diff
@@ -45,6 +45,9 @@ class TestSuite:
     server for this suite, the client program to execute individual
     tests and other suite properties. The server is started once per
     suite."""
+
+    DEFAULT_TEST_RETRIES_COUNT = 3
```

Comment:
Can it be useful to override this with an env variable?

Reply:
I think yes but if we added a special option to test-run (--retry-count). IMHO, at this moment the option is redundant.

Reply:
I would enter to Yaroslav here: I think that it is premature at least and doubt that we'll need it at all.